examples/set-namespace/.expected/diff.patch

```diff
@@ -0,0 +1,38 @@
+diff --git a/configs/example-config.yaml b/configs/example-config.yaml
+index 1c43349..2460ffb 100644
+--- a/configs/example-config.yaml
++++ b/configs/example-config.yaml
+@@ -2,6 +2,7 @@ apiVersion: v1
+ kind: ConfigMap
+ metadata:
+   name: the-map
++  namespace: example-ns
+ data:
+   some-key: some-value
+ ---
+@@ -9,7 +10,7 @@ apiVersion: v1
+ kind: Service
+ metadata:
+   name: the-service
+-  namespace: the-namespace
++  namespace: example-ns
+ spec:
+   ports:
+   - name: etcd-server-ssl
+diff --git a/functions/fn-config.yaml b/functions/fn-config.yaml
+deleted file mode 100644
+index 47eab5e..0000000
+--- a/functions/fn-config.yaml
++++ /dev/null
+@@ -1,11 +0,0 @@
+-apiVersion: v1
```

Comment:
what's with this diff?

Reply:
kpt will remove the fn-config.yaml after running. I am not sure is this expected behacior.